backend/src/utils/normaliser/skills.ts

```diff
@@ -0,0 +1,8 @@
+import { IUserSkill } from 'interfaces/Ientities/IUserSkill';
+
+const extractCommonUserSkillInfo = (skills: IUserSkill[]) =>
+  skills.map((uSkill) => {
+    return { score: uSkill.score, skill: uSkill.skill };
```

Comment:
also you can  `.map(uSkill) => ( {score: ... , skill: ...} )`

Reply:
I HAVEN'T EVER HEARD ABOUT THAT (ONE LINE OBJECT RETURN). THANKS 😆 😆 😆